integration/spark2/src/test/scala/org/apache/spark/carbondata/TestStreamingTableOperation.scala

```diff
@@ -2327,7 +2371,7 @@ class TestStreamingTableOperation extends QueryTest with BeforeAndAfterAll {
             .load()
 
           // Write data from socket stream to carbondata file
-          qry = readSocketDF.writeStream
+          qry = readSocketDF.repartition(2).writeStream
```

Comment:
Is this the place to generate empty rows to insert to streaming file?

Reply:
fixed